src/components/TaskRunDetailsView/tabs/TaskRunDetailsTab.tsx

```diff
@@ -107,6 +110,12 @@ const TaskRunDetailsTab: React.FC<TaskRunDetailsTabProps> = ({ taskRun, error })
                     </DescriptionListDescription>
                   </DescriptionListGroup>
                 )}
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Description</DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {taskRun.status?.taskSpec?.description || ''}
```

Comment:
nit:

 In pipelinerun details page, we use `-` as a fallback value for the fields that don't have any value.
```suggestion
                    {taskRun.status?.taskSpec?.description || '-'}
```

Reply:
updated